jstests/ssl/x509/mkcert.py

```diff
@@ -545,11 +621,12 @@ def process_cert(cert):
         process_client_multivalue_rdn(cert)
         return
 
-    if cert['name'] == 'ecdsa-ca.pem':
+    if cert['name'] in ['ecdsa-ca.pem', 'ecdsa-ca-ocsp.pem']:
         process_ecdsa_ca(cert)
         return
 
-    if cert['name'] in ['ecdsa-client.pem', 'ecdsa-server.pem']:
+    if cert['name'] in ['ecdsa-client.pem', 'ecdsa-server.pem', 'ecdsa-server-ocsp.pem',
```

Comment:
I think we can generalize this to search and see if the cert['name'] is prefixed with ecdsa, something like
`cert['name'].startsWith('ecdsa'):` but @sgolemon may have a stronger opinion on this.

Reply:
I'd lean to keeping it explicit, but how about we add `tags: [ ecdsa, ocsp, responder ]` type field to the YAML definition?  Then we can fix the slightly-too-magic `if 'responder' in cert['name']` and `if 'ocsp' in cert['name']:` checks above as well.

Reply:
I've kept it explicit and also added a "tags" field and am now using that instead of the slightly-too-magic names :)